src/server/game/Globals/ObjectMgr.h

```diff
@@ -1347,6 +1347,8 @@ class ObjectMgr
     void LoadFactionChangeSpells();
     void LoadFactionChangeTitles();
 
+    bool IsTransportMap(uint32 mapId) const { return _transportMaps.count(mapId) != 0; }
```

Comment:
please remember using `[[nodiscard]]` where applicable (here and in other changes too)